x-pack/plugins/ml/public/application/timeseriesexplorer/components/plot_function_controls/plot_function_controls.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { EuiFlexItem, EuiFormRow, EuiSelect } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+const plotByFunctionOptions = [
+  {
+    value: 'mean',
+    text: i18n.translate('xpack.ml.timeSeriesExplorer.plotByAvgOptionLabel', {
+      defaultMessage: 'mean',
+    }),
+  },
+  {
+    value: 'min',
+    text: i18n.translate('xpack.ml.timeSeriesExplorer.plotByMinOptionLabel', {
+      defaultMessage: 'min',
+    }),
+  },
+  {
+    value: 'max',
+    text: i18n.translate('xpack.ml.timeSeriesExplorer.plotByMaxOptionLabel', {
+      defaultMessage: 'max',
+    }),
+  },
+];
+export const PlotByFunctionControls = ({
+  functionDescription,
+  setFunctionDescription,
+}: {
+  functionDescription: undefined | string;
+  setFunctionDescription: (func: string) => void;
+}) => {
+  if (functionDescription === undefined) return null;
+  return (
+    <EuiFlexItem style={{ textAlign: 'right' }} grow={false}>
```

Comment:
to avoid inline styles, you can use [EuiText](https://elastic.github.io/eui/#/display/text) with `textAlign` prop

Reply:
Updated here https://github.com/elastic/kibana/pull/81662/commits/a7e2482b474068eec863111aab585bbd84b75e40